packages/synthetix-main/contracts/interfaces/ILiquidationModule.sol

```diff
@@ -3,13 +3,19 @@ pragma solidity ^0.8.0;
 
 /// @title Liquidates the collateral for an account in a pool
 interface ILiquidationModule {
+    struct LiquidationData {
+        uint debtLiquidated;
+        uint collateralLiquidated;
+        uint amountRewarded;
+    }
+
     event Liquidation(
         uint128 indexed accountId,
         uint128 indexed poolId,
         address indexed collateralType,
-        uint debtLiquidated,
-        uint collateralLiquidated,
-        uint amountRewarded
+        LiquidationData liquidationData,
+        uint liquidateAsAccountId,
+        address sender
     );
 
     event VaultLiquidation(
```

Comment:
this structure should be updated to use the same `LiquidationData` structure as well